_jugs/ParisJug.md

```diff
@@ -0,0 +1,10 @@
+---
+name:     "Paris Java Users Group"
```

Comment:
Users or User ?

Reply:
Well, if you have more than 1 user... 😉
I'm not English native, but I think the correct form should be in plural. 
For example, [here, in the JCP you can see the list of Java Users Groups](https://jcp.org/en/participation/JUG_list) where clearly we have 2 plurals, for users + groups

Reply:
yeah i think both are ok, but it's just to be consistent with our website/twitter/etc...